tools/dep_updaters/update-test426-fixtures.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+TARGET_DIR="$(dirname "$0")/../../test/fixtures/test426"
+README="$(dirname "$0")/../../test/test426/README.md"
+TARBALL_URL=$(curl -fsIo /dev/null -w '%header{Location}' https://github.com/tc39/source-map-tests/archive/HEAD.tar.gz)
+SHA=$(basename "$TARBALL_URL")
+
+TMP_DIR="$(mktemp -d)"
+curl -f "$TARBALL_URL" | tar -xzf - -C "$TMP_DIR"
+
+rsync -a --delete "$TMP_DIR"/source-map-tests-"$SHA"/ "$TARGET_DIR"/
```

Comment:
```suggestion
rsync -a --delete "$TMP_DIR/source-map-tests-$SHA/" "$TARGET_DIR/"
```
AFAICT we're not using `rsync` anywhere, so that's a new dependency. Have you tried instead:
```suggestion
rm -rf "$TARGET_DIR"
curl -f "$TARBALL_URL" | tar -xz --strip-components 1 -C "$TARGET_DIR"
```

Reply:
Need to `mkdir` after removing the directory as `curl` won't create it, but otherwise, that seems to work.